ethereum_history_api/circuits/lib/src/merkle_patricia_proofs/proof.nr

```diff
@@ -24,7 +24,7 @@ pub fn leaf_get_len<MAX_LEN>(leaf: Leaf<MAX_LEN>) -> u32 {
 }
 
 pub fn assert_merkle_proof<MAX_KEY_LEN, MAX_VALUE_LEN, MAX_DEPTH, MAX_LEAF_LEN>(
-    key: Key<MAX_KEY_LEN>,
+    _key: Key<MAX_KEY_LEN>,
```

Comment:
Why this `key` is not used? Maybe remove? This method is only using proof leaf and only leaf second element, IMO name `assert_merkle_proof` is misleading

Reply:
This is related to other ticket.